src/Broadcast.js

```diff
@@ -28,6 +28,26 @@ class Broadcast {
     });
   }
 
+  static atExcept(excludes, source, message, wrapWidth, useColor, formatter) {
```

Comment:
My rationale was that it reads more fluidly to be like, at, _Except_ (the thing modified by the word "except"), rest of arguments. That said, it does make sense to keep a similar signature, so I'll go ahead and modify it.